transformers4rec/torch/masking.py

```diff
@@ -402,6 +458,34 @@ def _compute_masked_targets(self, item_ids: torch.Tensor, training=False) -> Mas
 
         return MaskingInfo(mask_labels, labels)
 
+    @docstring_parameter(flags_parameters_docstrings=TRAINING_TESTING_FLAGS_DOCSTRING)
+    def apply_mask_to_inputs(
+        self, inputs: torch.Tensor, schema: torch.Tensor, training=False, testing=False
```

Comment:
Rename `schema` to `mask_schema` too as you did above, as `schema` is a "reserved" word in Merlin and here the schema is a tensor with the mask, which might confuse the user

Reply:
I renamed it to `mask_schema` in this specific method. But there are a set of methods/classes in `masking`  (not part of this PR scope) that uses the name `schema` instead of `mask_schema` so perhaps we can have a followup task that standardizes the masking code to use `mask_schema` instead of `schema` to refer to the boolean mask vector.